app/src/main/java/im/koala/bcsd/ui/login/LoginActivity.kt

```diff
@@ -0,0 +1,496 @@
+package im.koala.bcsd.ui.login
+
+import android.content.Intent
+import android.os.Bundle
+import androidx.activity.ComponentActivity
+import androidx.activity.compose.setContent
+import androidx.compose.foundation.Image
+import androidx.compose.foundation.background
+import androidx.compose.foundation.border
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.Row
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.height
+import androidx.compose.foundation.layout.size
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.text.KeyboardOptions
+import androidx.compose.material.Button
+import androidx.compose.material.ButtonDefaults
+import androidx.compose.material.Divider
+import androidx.compose.material.MaterialTheme
+import androidx.compose.material.OutlinedTextField
+import androidx.compose.material.Surface
+import androidx.compose.material.Switch
+import androidx.compose.material.SwitchDefaults
+import androidx.compose.material.Text
+import androidx.compose.material.TextButton
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.RectangleShape
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.text.input.ImeAction
+import androidx.compose.ui.text.input.KeyboardType
+import androidx.compose.ui.text.input.TextFieldValue
+import androidx.compose.ui.unit.dp
+import androidx.constraintlayout.compose.ConstraintLayout
+import androidx.constraintlayout.compose.Dimension
+import im.koala.bcsd.R
+import im.koala.bcsd.ui.findid.FindIdActivity
+import im.koala.bcsd.ui.findpassword.FindPasswordActivity
+import im.koala.bcsd.ui.join.JoinActivity
+import im.koala.bcsd.ui.theme.Black
+import im.koala.bcsd.ui.theme.Gray
+import im.koala.bcsd.ui.theme.GrayBorder
+import im.koala.bcsd.ui.theme.GrayDisabled
+import im.koala.bcsd.ui.theme.GrayNormal
+import im.koala.bcsd.ui.theme.Green
+import im.koala.bcsd.ui.theme.KoalaTheme
+import im.koala.bcsd.ui.theme.White
+import im.koala.bcsd.ui.theme.Yellow
+import im.koala.bcsd.ui.theme.Yellow2
+
+class LoginActivity : ComponentActivity() {
+    override fun onCreate(savedInstanceState: Bundle?) {
+        super.onCreate(savedInstanceState)
+        setContent {
+            KoalaTheme {
+                // A surface container using the 'background' color from the theme
+                Surface(color = MaterialTheme.colors.background) {
+                    constraintLayoutContent()
+                }
+            }
+        }
+    }
+}
+
+@Composable
+fun constraintLayoutContent() {
```

Comment:
Compose method는 Pascal convention을 사용합니다.
```suggestion
fun ConstraintLayoutContent() {
```

Reply:
ConstraintLayoutContent 보다는 LoginScreen와 같이 의미 있는 메소드 명으로 변경하면 좋을 것 같습니다.

Reply:
적용하였습니다.